contrib/lang/c-c++/packages.el

```diff
@@ -34,7 +34,14 @@ which require an initialization must be listed explicitly in the list.")
       (semantic-mode 1)
       (c-toggle-auto-newline 1)
       (setq srecode-map-save-file (concat spacemacs-cache-directory "srecode-map.el"))
-      (setq semanticdb-default-save-directory (concat spacemacs-cache-directory "semanticdb/")))))
+      (setq semanticdb-default-save-directory (concat spacemacs-cache-directory "semanticdb/"))
+      (evil-leader/set-key-for-mode 'c-mode
+        "mgo" 'projectile-find-other-file
+        "mgO" 'projectile-find-other-file-other-window)
+      (evil-leader/set-key-for-mode 'c++-mode
```

Comment:
This should be moved into `init-projectile` and `init-helm-projectile`, since you can switch not only in C/C++ but in Ruby (i.e. Gemfile <-> Gemfile.lock) and Javascript as well.